tests/exporters/tflite/test_export_cli.py

```diff
@@ -300,7 +300,7 @@ def test_trust_remote_code(self):
 
         with TemporaryDirectory() as tmpdirname:
             out = subprocess.run(
-                f"python3 -m optimum.exporters.tflite --trust-remote-code --model fxmarty/tiny-testing-gpt2-remote-code --task text-generation {tmpdirname}",
+                f"python3 -m optimum.exporters.tflite --trust-remote-code --model optimum-internal-testing/tiny-testing-gpt2-remote-code --task text-generation {tmpdirname}",
```

Comment:
both `optimum-internal-testing/tiny-testing-gpt2-remote-code` and `fxmarty/tiny-testing-gpt2-remote-code` were still using a deprecated (now removed) class from transformers, fixed in https://huggingface.co/optimum-internal-testing/tiny-testing-gpt2-remote-code/commit/177a59ecdc1fcbc8e039b19b7a47f5251d4132e0